rollingpin/deploy.py

```diff
@@ -169,6 +177,10 @@ def signal_handler(sig, _):
                             except KeyError:
                                 raise ComponentNotBuiltError(component)
                             deploy_command.append(deploy_ref)
+
+                    # wait on ready command
+                    if not self.dangerously_fast:
```

Comment:
Is there any harm doing a little health check of whether components are ready whether you restart or not?